### PR TITLE
New chat improvements

### PR DESCRIPTION
**Pull request by philipperolet**
Docs sorting, /retrieve, loading message, debump chunks
Comments below

### CODE REVIEW

File: front/styles/global.css [hunk 1]
```diff
@@ -11,3 +11,17 @@ main {
 #__next {
   @apply h-full;
 }
+
+.loading-dots::after {
```
Comment (philipperolet) [inline front/styles/global.css:15]: not sure that's where to put it -- and at this time just a suggestion

**Comment by philipperolet** [comment]
UI message: "Computed query: {}. Retrieving docs..." with animated dots

Tough to put a "Computing query" since we don't know whether the 
We could do a "Thinking..."--it would insist on what the setLoading(true) does (soft blinking dust logo); not sure it is needed, TBD

**Comment by philipperolet** [comment]
Debump chunks back to 16: did it here as suggestion. IMHO It's TMI for the model, currently seems to degrade answer quality.
Once we build an eval we can put metrics on that and make an informed decision

File: front/styles/global.css (hunk 1 above)
Comment (spolu) [reply front/styles/global.css:15]: that's a fair place

**Review by spolu — APPROVED** [review]
Looks great

**Pull request merged**